elotl/utils/morphology.py

```diff
@@ -223,24 +224,49 @@ class Analyser(object):
 
 	Parameters
 	----------
+	lang_code: str
+		ISO-639-3 code for the language variety for which you want to load and use 
+		the morphological analyzer. If not supported, 
 	tokeniser: function
 		A tokenisation function, if none is provided a default tokeniser, _tokenise()
 		is used which is based on regular expressions.
 
 	"""
-	def __init__(self, tokeniser=None):
+	def __init__(self, lang_code, tokeniser=None):
```

Comment:
Conviene usar typing para tener más claridad de que espera el constructor:

```suggestion
	def __init__(self, lang_code: str, tokeniser=None):
```

Reply:
Esto lo agregaría en todos lados donde sea posible, puedes usar un LLM pará que te haga paro :D